pkg/trie/triedb/triedb.go

```diff
@@ -685,4 +699,166 @@ func (t *TrieDB) lookupNode(hash common.Hash) (StorageHandle, error) {
 	}), nil
 }
 
+func (t *TrieDB) commit() error {
+	log.Printf("Committing trie changes to db")
```

Comment:
Should we use our internal logger here instead?

Reply:
I took a similar approach than @timwu20 with https://github.com/ChainSafe/gossamer/pull/4015

Reply:
I would suggest using `logger.Debug("Committing trie changes to db")` so we can have it logged following our logging style

Reply:
Changed to use pkg level logger! thks